src/pages/Account/Venue/VenueMapEditor/interfaces.ts

```diff
@@ -1,5 +1,5 @@
 export interface DragItem {
-  id: string;
+  itemId: number;
```

Comment:
If this is saved to DB, would it be somehow converted on read or write to `string`?